packages/ses/src/error/note-log-args.js

```diff
@@ -7,36 +8,37 @@ const { freeze } = Object;
 const { isSafeInteger } = Number;
 
 /**
+ * @template K,V
  * @typedef {object} DoublyLinkedCell
  * A cell of a doubly-linked ring, i.e., a doubly-linked circular list.
  * DoublyLinkedCells are not frozen, and so should be closely encapsulated by
  * any abstraction that uses them.
- * @property {DoublyLinkedCell} next
- * @property {DoublyLinkedCell} prev
- * @property {object} key
- * @property {any} value;
+ * @property {DoublyLinkedCell<K, V>} next
+ * @property {DoublyLinkedCell<K, V>} prev
+ * @property {WeakMap<object, V>} keyValue
```

Comment:
I assume `K` is for key?

```suggestion
 * @property {WeakMap<K, V>} keyValue
```

Reply:
Can't do this, because...

Reply:
Yeah given the comment on `set` below, We probably should rename this to `data` with a type of `T` type param solely defined by the user of the linked list.